### PR TITLE
Added basic parsing validation of the swagger yaml.

### PR DESCRIPTION
**Pull request by kshakir**
Removed invalid YAML content.
Also while adding the yaml library, cleaned up the build.sbt IntelliJ IDEA warnings.

### CODE REVIEW

**Comment by kshakir** [comment]
Hoping @geoffjentry can provide some insight here as to how to get this pack up and running. Perhaps @jacarey can be second reviewer if available.

File: src/main/resources/swagger/cromwell.yaml [hunk 1]
```diff
@@ -301,99 +301,6 @@ paths:
...
-                "sc_test.do_scatter.count_file": ["/home/jdoe/cromwell/cromwell-executions/sc_test/6efd83b7-39df-47d7-bb2d-96ebb9f093a2/call-do_scatter/shard-0/output.txt", "/home/jdoe/cromwell/cromwell-executions/sc_test/6efd83b7-39df-47d7-bb2d-96ebb9f093a2/call-do_scatter/shard-1/output.txt", "/home/jdoe/cromwell/cromwell-executions/sc_test/6efd83b7-39df-47d7-bb2d-96ebb9f093a2/call-do_scatter/shard-2/output.txt", "/home/jdoe/cromwell/cromwell-executions/sc_test/6efd83b7-39df-47d7-bb2d-96ebb9f093a2/call-do_scatter/shard-3/output.txt"]
-              },
-              "id": "6efd83b7-39df-47d7-bb2d-96ebb9f093a2",
-              "inputs": {
-                "sc_test.do_prepare.input_file": "/home/jdoe/cromwell/11.txt"
-              },
-              "submission": "2015-09-29T14:06:32.120-04:00",
-              "status": "Succeeded",
-              "start": "2015-09-29T14:06:32.120-04:00"
-            }
```
Comment (mcovarr) [inline src/main/resources/swagger/cromwell.yaml]: Urg, that's kind of unfortunate.  I had a feeling this was the cause of offense to YAML linters.  :frowning:

**Comment by geoffjentry** [comment]
With the caveat that I don't know where we're supposed to stick it, :+1:

**Comment by kshakir** [comment]
Additional commit: example was missing an indentation. Fixed.

**Comment by mcovarr** [comment]
Thanks @kshakir, that's awesome!  Swagger's expressive power didn't seem able to capture the shape of the returned data, so having the example restored will I'm sure be much appreciated by Blue.  :smile:

**Comment by cjllanwarne** [comment]
Great! :+1:

**Pull request merged**